driver/config/config.go

```diff
@@ -1029,6 +1031,14 @@ func (p *Config) SessionWhoAmIAAL() string {
 	return p.p.String(ViperKeySessionWhoAmIAAL)
 }
 
+func (p *Config) SessionWhoAmIRefresh() bool {
+	return p.p.Bool(ViperKeySessionWhoAmIRefresh)
+}
+
+func (p *Config) SessionRefreshTimeWindow() time.Duration {
+	return p.p.DurationF(ViperKeySessionRefreshTimeWindow, p.SessionLifespan()/2)
```

Comment:
A particular reason for setting this to half of lifespan?

Reply:
I think that @harnash set this so that the default value wouldn't allow session refresh on every request. I think we can leave the default to 0 and just cover this in the docs